app/models/git_label.py

```
@@ -1,46 +1,50 @@
-        table_description = (
-            "Table for storing git hosting service configurations per user"
-        )
-
-    def __str__(self):
-        return f"GitLabel(id={self.id}, user_id={self.user_id}, label={self.label}, git_hosting={self.git_hosting})"
 
-    def __repr__(self):
-        return self.__str__()
\ No newline at end of file
+class GitLabel(Model):
+	"""
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)